modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -902,43 +907,45 @@ public void setParameterExampleValue(CodegenParameter p) {
             type = p.dataType;
         }
 
-        if ("String".equalsIgnoreCase(type) || "str".equalsIgnoreCase(type)) {
+        if (type != null) {
+          if ("String".equalsIgnoreCase(type) || "str".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = p.paramName + "_example";
+              example = p.paramName + "_example";
             }
             example = "'" + escapeText(example) + "'";
-        } else if ("Integer".equals(type) || "int".equals(type)) {
+          } else if ("Integer".equals(type) || "int".equals(type)) {
             if (example == null) {
-                example = "56";
+              example = "56";
             }
-        } else if ("Float".equalsIgnoreCase(type) || "Double".equalsIgnoreCase(type)) {
+          } else if ("Float".equalsIgnoreCase(type) || "Double".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = "3.4";
+              example = "3.4";
             }
-        } else if ("BOOLEAN".equalsIgnoreCase(type) || "bool".equalsIgnoreCase(type)) {
+          } else if ("BOOLEAN".equalsIgnoreCase(type) || "bool".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = "True";
+              example = "True";
             }
-        } else if ("file".equalsIgnoreCase(type)) {
+          } else if ("file".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = "/path/to/file";
+              example = "/path/to/file";
             }
             example = "'" + escapeText(example) + "'";
-        } else if ("Date".equalsIgnoreCase(type)) {
+          } else if ("Date".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = "2013-10-20";
+              example = "2013-10-20";
             }
             example = "'" + escapeText(example) + "'";
-        } else if ("DateTime".equalsIgnoreCase(type)) {
+          } else if ("DateTime".equalsIgnoreCase(type)) {
             if (example == null) {
-                example = "2013-10-20T19:20:30+01:00";
+              example = "2013-10-20T19:20:30+01:00";
             }
             example = "'" + escapeText(example) + "'";
-        } else if (!languageSpecificPrimitives.contains(type)) {
+          } else if (!languageSpecificPrimitives.contains(type)) {
             // type is a model class, e.g. User
             example = this.packageName + "." + type + "()";
-        } else {
+          } else {
             LOGGER.warn("Type " + type + " not handled properly in setParameterExampleValue");
+          }
```

Comment:
FYI. Looks like these changes are not indented properly (2-space vs 4-space). I can fix it later after merging this PR.

Reply:
Noted. Thank you!